File: app/src/main/java/com/example/whattodo/ActivityComent.java
package com.example.whattodo;

import android.content.Intent;
import android.os.Bundle;
import android.os.PersistableBundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class ActivityComent extends AppCompatActivity {

    @Override
    public void onCreate( Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_coment);
        getSupportFragmentManager().beginTransaction().replace(R.id.frame_coment, new ComentFragment()).commit();

    }
}

File: app/src/main/java/com/example/whattodo/ActivityComent.kt
package com.example.whattodo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class ActivityComent : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_coment)
        supportFragmentManager.beginTransaction().replace(R.id.frame_coment, ComentFragment()).commit()
    }
}